engine/evarutil.mli

```diff
@@ -42,6 +42,7 @@ val new_evar :
 
 val new_pure_evar :
   ?src:Evar_kinds.t Loc.located -> ?filter:Filter.t ->
+  ?identity:EConstr.t list ->
```

Comment:
Given that this function is exported, I think a comment should be present and point to `Evd.Identity`